hydrant/adapters/sites/skagit.py

```diff
@@ -4,6 +4,34 @@
 from hydrant.models.service_request import ServiceRequest
 
 
+def labcorp_code_lookup(labcorp_code):
+    """Sites such as Skagit send all labs to Labcorp - lookup keyed values
+
+    Given flat CSV files, need to map a code from a known lab system to
+    a FHIR Service Request.
+
+    Extend with additional codes as needed.  Look up values at top of
+    an existing page such as:
+    https://www.labcorp.com/tests/733727/pain-management-screening-profile-11-drugs-urine-pmp-11s
```

Comment:
Nice find! Looks like these are LOINC under the hood- should we just use LOINC as the system instead?

Reply:
they are panels - groups of 11 and 13 tests, each of which have their own LOINC test and result code.  the panel may have a similar LOINC panel, but we haven't found it.

at some point, we may chose to capture all 11 or 13 tests codings, but at this time, that's overkill.  will be interesting to see what a real EMR provides.